### ISSUE TITLE
Status of torcx in fedora/redhat coreos?

### ISSUE THREAD

**Issue by kamalmarhubi** [coreos/torcx#123]
I built a bit of tooling to package things up for torcx. Is there any clarity yet on whether torcx has a place in coreos' future? I'd like to either polish that up and share, or switch to whatever the new OS will use for the purpose.

**Comment by bgilbert** [comment]
Hi @kamalmarhubi!  torcx is unlikely to make it into either Fedora CoreOS or Red Hat CoreOS.

Fedora CoreOS will likely have some mechanism for switching to a different version of a container runtime or kubelet.  We're not sure yet whether this will be system containers or systemd portable services (https://github.com/coreos/fedora-coreos-tracker/issues/37) or rpm-ostree package layering, but it will almost certainly not be torcx.

Red Hat CoreOS is unlikely to provide such a mechanism at all, since a RHCOS release will be tuned for the requirements of a particular version of OpenShift.

**Comment by lucab** [comment]
Indeed. [rpm-ostres](https://github.com/projectatomic/rpm-ostree) provides a superset of torcx features.
